integrations/sonarqube/tests/fixtures.py

```diff
@@ -0,0 +1,244 @@
+from typing import Any
+
+PURE_PROJECTS: list[dict[str, Any]] = [
+    {
+        "key": "project-key-1",
+        "name": "Project Name 1",
+        "qualifier": "TRK",
+        "visibility": "public",
+        "lastAnalysisDate": "2017-03-01T11:39:03+0300",
+        "revision": "cfb82f55c6ef32e61828c4cb3db2da12795fd767",
+        "managed": False,
+    },
```

Comment:
ahh maybe this is the "raw" data of the "deprecated projects? if so ignore my previous comment